abstraction_scores.ipynb

```diff
@@ -19,35 +19,13 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[nltk_data] Downloading package punkt to /home/jovyan/nltk_data...\n",
-      "[nltk_data]   Package punkt is already up-to-date!\n",
-      "[nltk_data] Downloading package stopwords to /home/jovyan/nltk_data...\n",
-      "[nltk_data]   Package stopwords is already up-to-date!\n",
-      "[nltk_data] Downloading package wordnet to /home/jovyan/nltk_data...\n",
-      "[nltk_data]   Package wordnet is already up-to-date!\n"
-     ]
-    },
-    {
-     "data": {
-      "text/plain": [
-       "True"
-      ]
-     },
-     "execution_count": 1,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "import nltk\n",
-    "nltk.download(\"punkt\")\n",
-    "nltk.download(\"stopwords\")\n",
-    "nltk.download(\"wordnet\")"
+    "# tjm: commented these out because they are all \"already up-to-date\" and give error when offline\n",
```

Comment:
There shouldn't be an error. It should be a warning of the same message. Either way, since we will be transitioning to plain Python, changes in abstraction_scores.ipynb can be ignored.

Reply:
OK to ignore. I assume I don't have to do anything.